lib/ancestry/has_ancestry.rb

```diff
@@ -30,6 +30,13 @@ def has_ancestry options = {}
       validates_format_of self.ancestry_column, :with => derive_ancestry_pattern(options[:primary_key_format]), :allow_nil => true
       extend Ancestry::MaterializedPath
 
+      @@default_update_strategy = :sql
+      def self.default_update_strategy
+        @@default_update_strategy
+      end
```

Comment:
wait, why did you put this here?
I thought it would be in Ancestry module and usable as a global for all

That way in environment.rb we could set this to sql for all tests.

I liked how you had this as :ruby before.

tests: use sql
default for everyone else: sql